### DISCUSSION TITLE
What is the default username and pasword for Gke get started

### DISCUSSION THREAD

**Discussion by abygonz** [eclipse-theia/theia-cloud#181] [Q&A]
I so proud with this product, im triying to use GKE started, the servie is up but i dont know what user and password por page.

**Comment by jfaltermeier** [comment]
Hi, 

For the getting started example we are creating two test users called `foo` and `bar`. The passwords are the same as the username. 

By the way, when running `terraform apply` for the GKE getting started you have to specify a password for the keycloak admin user. 

```
var.keycloak_admin_password
  Keycloak Admin Password

  Enter a value: 
```

You can use this to log into the keycloak admin console and create further users, change the password the password for existing ones, and so on.

**Comment by macurandb** [comment]
Hi @jfaltermeier, 
I installed the one in GKE, but when I tried to log in I got the error message "Invalid username or password". I tried with the user Admin or Admin but I got the same error, I entered the correct password. Could you please suggest a solution?

**Reply by jfaltermeier** [reply]
The GKE Getting started has two users, `foo` and `bar`. Passwords for those sample users are their usernames: https://github.com/eclipsesource/theia-cloud/blob/685e1eca2f3b78b2f4882a54f03ffd6076f4a1e9/terraform/configurations/gke_getting_started/gke_getting_started.tf#L91

**Reply by macurandb** [reply]
I am sorry for not making myself understand but what I want is to be able to log in to the administrator to configure the system and I have not been able to do it with the admin user, maybe I have lost something Can you give me some suggestions?
